web-components/src/components/modal/BasketPutModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { RegenModalProps } from '../modal';
+import { FormModalTemplate } from './FormModalTemplate';
+import { BasketPutForm, BasketPutProps } from '../form/BasketPutForm';
+
+interface BasketPutModalProps extends RegenModalProps, BasketPutProps {}
+
+const BasketPutModal: React.FC<BasketPutModalProps> = ({
+  basketOptions,
+  batchDenom,
+  availableTradableAmount,
+  open,
+  onClose,
+  onSubmit,
+}) => (
+  <FormModalTemplate title="Put in basket" open={open} onClose={onClose}>
```

Comment:
@mhagel you can just re-use this same FormModalTemplate component for https://github.com/regen-network/regen-registry/issues/816